machine_learning/lstm/lstm_prediction.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
```

Comment:
It would be helpful if this file defined what LSTM is.  Or at least pointed to an article like https://en.wikipedia.org/wiki/Long_short-term_memory to so than readers do not need to guess.

The only function in this submission is `clean_data()` so is this the submission of an example (as the PR title says) of how to use Python packages or is there an algorithm here?  See CONTRIBUTING.md's [What is an algorithm?](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md#what-is-an-algorithm)

Reply:
This is an example of LSTM stock prediction using Keras,Which is a deep-learning library